atlasdb-tests-shared/src/main/java/com/palantir/timelock/paxos/InMemoryTimelockExtension.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.timelock.paxos;
+
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+public final class InMemoryTimelockExtension extends AbstractInMemoryTimelockExtension
+        implements BeforeEachCallback, AfterEachCallback {
+
+    @Override
+    public void beforeEach(ExtensionContext var1) {
+        setup();
+    }
+
+    @Override
+    public void afterEach(ExtensionContext var1) {
+        tearDown();
```

Comment:
Do you need to call `super.afterEach()` as well to make sure we call `close` and trigger the shutdown?

Reply:
ah, nvm, just got confused here!